test/endtoend/melchett.test.ts

```diff
@@ -179,6 +179,47 @@ describe('melchett client', () => {
     });
   });
 
+  describe('circuit breakers - multiple clients', () => {
+    const config = {
+      name: 'test',
+      circuitBreaker: {
+        errorThresholdPercentage: 10,
+        resetTimeout: 1000,
+        rollingCountTimeout: 6000,
+        volumeThreshold: 50
+      }
+    };
+
+    beforeAll(async () => {
+      nock('http://testurl.com')
+        .persist()
+        .get('/x')
+        .reply(500, undefined);
+    });
+
+    it('should NOT fire the open circuit breaker when running multiple clients', async () => {
```

Comment:
Would this ever trip the circuit breaker anyway? `volumeThreshold` is at 50 so I don't think the multiple clients is really having any affect here on whether or not it trips, as it only sends three requests.

Reply:
good point i'll remove the `volumeThreshold` 👍